styles/google/catppuccin.user.css

```diff
@@ -0,0 +1,1093 @@
+/* ==UserStyle==
+@name           Google Catppuccin Less
+@namespace      github.com/GitMuslim/userstyles/styles/google
```

Comment:
Should be `github.com/catppuccin/userstyles/styles/google`.

Reply:
Forgot to change this